Portfolio/src/views/HomeView.vue

```diff
@@ -1,134 +1,144 @@
-    width: 100%;
-    height: 65vh;
-    font-family: "Raleway";
+    font-family: 'Raleway';
     text-transform: uppercase;
-}
+    letter-spacing: -2px;
 
+}
 
-.homepage-leftcontainer {
+.home-page-left-container {
     display: flex;
     flex-direction: column;
     color: #1e1e1e;
     font-size: 20px;
-    padding: 0 4rem;
+    margin-top: 5rem;
 
 }
 
-.home-smallName,
-.homepage-title {
+.home-page-small-name,
+.home-page-title {
     margin: 0;
 }
 
-.homepage-title {
+.home-page-title {
     font-size: 100px;
-    width: 20rem;
-    letter-spacing: -2px;
-
+    width: 15rem;
 }
 
-.homepage-rightcontainer {
+.home-page-right-container {
     display: flex;
     flex-direction: column;
     font-size: 50px;
     color: #1e1e1e;
-    letter-spacing: -2px;
     align-self: flex-end;
-    padding: 0 4rem;
+    text-align: right;
+    margin-top: 5rem;
+
 }
 
-.home-firstName,
-.home-lastName {
+.home-page-firstName,
+.home-page-lastName {
     margin: 0;
 }
 
 @media (min-width: 280px) and (max-width: 767px) {
-    .homepage-leftcontainer {
-        margin: 4rem 2rem;
+    .home-page-left-container {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+    }
+
+    .home-page-title {
+        font-size: 40px;
+        justify-content: center;
+        align-items: center;
+        text-align: center;
     }
 
-    .homepage-title {
-        font-size: 45px;
+    .home-page-rightcontainer {
+        font-size: 20px;
+        justify-content: center;
+        align-items: center;
     }
 
-    .homepage-rightcontainer {
-        font-size: 25px;
+    .home-page-small-name {
+        font-size: 20px;
     }
 }
 
 @media (min-width: 768px) and (max-width: 1024px) {
-    .homepage-leftcontainer {
-        margin: 7rem 4rem;
+    .home-wrapper {
+        padding: 10px;
     }
 
-    .homepage-title {
+    .home-page-title {
         font-size: 60px;
-        width: 20rem;
     }
 
-
-    .homepage-rightcontainer {
+    .home-page-right-container {
         font-size: 32px;
-        margin: 33rem 4rem 0 0;
     }
-
 }
 
 @media (min-width: 1025px) and (max-width: 1280px) {
-    .homepage-leftcontainer {
-        margin: 7rem 6rem;
-    }
 
-    .homepage-title {
+    .home-page-title {
         font-size: 65px;
         width: 20rem;
     }
 
-    .homepage-rightcontainer {
+    .home-page-right-container {
         font-size: 33px;
-        margin: 25rem 6rem 0 0;
     }
 }
 
 @media (min-width: 1281px) and (max-width: 1899px) {
-    .homepage-leftcontainer {
-        margin: 7rem 6rem;
+
+    .home-page-left-container {
+        margin-top: 5rem;
     }
 
-    .homepage-title {
+    .home-page-title {
         font-size: 75px;
         width: 20rem;
     }
 
-    .homepage-rightcontainer {
+    .home-page-right-container {
         font-size: 40px;
-        margin: 26rem 6rem 0 0;
     }
 }
+
+@media (min-width: 1900px) {
+    .home-page-right-container {
+        margin-top: 15rem;
+    }
+
+}
```

Comment:
Extract the styles to their own file and import them to the component so that you can have separation of concerns and a more organized codebase

Reply:
Done